RecoEgamma/EgammaTools/src/EnergyScaleCorrection.cc

```diff
@@ -359,7 +361,22 @@ EnergyScaleCorrection::CorrectionCategory::CorrectionCategory(const std::string&
     r9Min_ = -1;
     r9Max_ = 0.94;
   };	
-  
+  // R9 region
+  p1 = category.find("-R9");
+  p2 = p1 + 1;
+  if(p1 != std::string::npos) {
+    p1 = category.find("_", p1);
+    p2 = category.find("_", p1 + 1);
+    r9Min_ = TString(category.substr(p1 + 1, p2 - p1 - 1)).Atof();
+    // If there is one value, just set lower bound
+    if (p2 != std::string::npos) {
+      p1 = p2;
+      p2 = category.find("-", p1);
+      r9Max_ = TString(category.substr(p1 + 1, p2 - p1 - 1)).Atof();
+      if(r9Max_>=1.0) r9Max_ = std::numeric_limits<float>::max();
+    }
+    //std::cout << r9Min_ << "\t" << r9Max_ << "\t" << category.substr(p1 + 1, p2 - p1 - 1) << std::endl;
```

Comment:
is this commented out code needed?
Please remove or add comments inline in the code why the commented out block is relevant